lib/Driver/ToolChains.cpp

```diff
@@ -1016,6 +1016,17 @@ static void getRuntimeLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
                           getPlatformNameForTriple(TC.getTriple()));
 }
 
+/// On Darwin we link fat libs in the root of the library path and on other OS
+/// targets we look up libraries in the target arch's subfolder.
```

Comment:
Nitpick: This comment doesn't really describe the function.

Reply:
Ah true. What I was thinking when I changed where we used it and not what the function does.